docs/getting-started.md

```diff
@@ -181,18 +181,40 @@ token.setAddress('owner', event.params.to)
 token.setU256('amount', event.params.tokens)
 ```
 
-There is also a global `Store` class which has a `set` method for setting the value(s) of a particular entity's attribute(s) in the store.
+There is also a global `store` class which has `set` and `get` methods for setting and getting the value(s) of a particular entity's attribute(s) in the store.
 
-It expects the name of an entity type, the id of the entity and the `Entity` itself.
+
+#### `store.set(entity: string, id: string, data: Entity)`
+
+`store.set` expects the name of an entity type, the id of the entity and the `Entity` itself.
 
 ##### Example
 ```typescript
   store.set('Token', tokenId, token)
 ```
 
-The eventHandlers functions return `void`. The only way that entities may be added to the The Graph is by calling `Store.set()`. `Store.set()` may be called multiple times in an event handler.
+The eventHandlers functions return `void`. The only way that entities may be added to the The Graph is by calling `store.set()`. `store.set()` may be called multiple times in an event handler.
+
+**Note** `store.set()` will only set the entity attributes that have explicitly been set on the `Entity`. Attributes which are not explicitly set, or unset by calling `Entity.unset(<attribute>)`, will not be overwritten.
+
+#### `store.get(entity: string, id: string)`
+
+You can use `store.get` to retreive information previously added with `store.set`.
+`store.get` expects entity type and the id of the entity.
+
+##### Example
+
+```javascript
+  store.set('Challenge', challengeId.toHex(), challenge)
+  let challenge2 = store.get('Challenge', challengeId.toHex())
+  let id = challenge2.getString("application")
+  let challenge2entity = new Entity()
```

Comment:
I find this example to be a little artificial and confusing — it would be better to use two completely separate entity types here (e.g. `Challenge` and `Registration` or something).